core/container.go

```diff
@@ -69,6 +70,8 @@ func (c *Container) GetContainerImage() (image string, err error) {
 		return "registry.opensuse.org/opensuse/tumbleweed:latest", nil
 	case XBPS:
 		return "ghcr.io/void-linux/void-linux:latest-full-x86_64", nil
+	case SWUPD:
+		return "docker.io/library/clearlinux", nil
```

Comment:
The documentation is very ambiguous in that regard, it specifies:
"The clearlinux:latest tag will point to clearlinux:base which will track toward the latest release version of the distribution."
But still, I think using the latest tag is a safe option to avoid ambiguity and make sure users are getting the most up to date version.